src/main/java/mate/academy/bookstore/controller/BookController.java

```diff
@@ -57,6 +61,7 @@ public BookDto updateBookById(@PathVariable Long id,
         return bookService.updateById(id, requestDto);
     }
 
+    @PreAuthorize("hasRole('ROLE_ADMIN')")
```

Comment:
Have you tested it? I can be wrong, but how I know "When using hasRole(), do not add the ROLE_ prefix, Spring Security add it, for example, hasRole("CUSTOMER")."